recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ test:
     - sksparse
     - sksparse.cholmod
   commands:
-    - nosetests -sv sksparse
+    - OPENBLAS_NUM_THREADS=1 nosetests -sv sksparse
```

Comment:
Just curious... I abolished `openblas` from my local builds due to similar time-out errors and some bad matrix manipulation results. When I asked upstream I got a "this is a feature and not a bug." So I decided to stick with `mkl` after that.